sandboxed-fs.js

```diff
@@ -4,11 +4,109 @@ const sandboxedFs = {};
 module.exports = sandboxedFs;
 
 const fs = require('fs');
```

Comment:
I'd prefer imports to be placed at the top of the file, right after `'use strict';`.